src/styles/variables.scss

```
@@ -0,0 +1,6 @@
+$tiffanyColor: #52d6c8;
```

It's not js, so use `-` to separate words
```suggestion
$c-tiffany: #52d6c8;
```